common/test/acceptance/tests/lms/test_lms_dashboard_search.py

```diff
@@ -0,0 +1,192 @@
+    EMAIL = 'student101@example.com'
+
+    STAFF_USERNAME = "STAFF_TESTER"
+    STAFF_EMAIL = "staff101@example.com"
+
+    TEST_INDEX_FILENAME = "test_root/index_file.dat"
+
+    def setUp(self):
+        """
+        Create the search page and courses to search.
+        """
+        # create test file in which index for this test will live
+        with open(self.TEST_INDEX_FILENAME, "w+") as index_file:
+            json.dump({}, index_file)
+
+        super(DashboardSearchTest, self).setUp()
+        self.dashboard = DashboardSearchPage(self.browser)
+
+        self.courses = {
+            'A' : {
+                'org': 'test_org',
+                'number': self.unique_id,
+                'run': 'test_run_A',
+                'display_name': 'Test Course A '
+            },
+            'B' : {
+                'org': 'test_org',
+                'number': self.unique_id,
+                'run': 'test_run_B',
+                'display_name': 'Test Course B '
+            },
+            'C' : {
+                'org': 'test_org',
+                'number': self.unique_id,
+                'run': 'test_run_C',
+                'display_name': 'Test Course C '
+            }
+        }
+
+        # generate course fixtures and outline pages
+        self.course_outlines = {}
+        self.course_fixtures = {}
+        for key, course_info in self.courses.iteritems():
+            course_outline = CourseOutlinePage(
+                self.browser,
+                course_info['org'],
+                course_info['number'],
+                course_info['run']
+            )
+
+            course_fix = CourseFixture(
+                course_info['org'],
+                course_info['number'],
+                course_info['run'],
+                course_info['display_name']
+            )
+
+            course_fix.add_children(
+                XBlockFixtureDesc('chapter', 'Section 1').add_children(
+                    XBlockFixtureDesc('sequential', 'Subsection 1').add_children(
+                        XBlockFixtureDesc('problem', 'dashboard search')
+                    )
+                )
+            ).add_children(
+                XBlockFixtureDesc('chapter', 'Section 2').add_children(
+                    XBlockFixtureDesc('sequential', 'Subsection 2')
+                )
+            ).install()
+
+            self.course_outlines[key] = course_outline
+            self.course_fixtures[key] = course_fix
+
+    def tearDown(self):
+        """
+        Remove index file
+        """
+        os.remove(self.TEST_INDEX_FILENAME)
+
+    def _auto_auth(self, username, email, staff):
+        """
+        Logout and login with given credentials.
+        """
+        LogoutPage(self.browser).visit()
+        AutoAuthPage(self.browser, username=username, email=email, staff=staff).visit()
+
+    def _studio_add_content(self, course_outline, html_content):
+        """
+        Add content on studio course page under specified section.
```

Comment:
Seems to always be the first section - seems okay to me, but doc string is now out of date